assets/ar/factuality_disinformation_harmful_content/subjectivity/Subjectivity_GPT4_ZeroShot_explain.py

```diff
@@ -0,0 +1,72 @@
+from llmebench.datasets import ThatiARDataset
+from llmebench.models import OpenAIModel
+from llmebench.tasks import SubjectivityTask
+
+
+def metadata():
+    return {
+        "author": "Arabic Language Technologies, QCRI, HBKU",
+        "model": "gpt-4-32k (version 0314)",
+        "description": "GPT4 32k tokens model hosted on Azure, using the ChatCompletion API. API version '2023-03-15-preview'. Uses an prompt specified in Arabic.",
+        "scores": {"Macro-F1": "0.725"},
+    }
+
+
+def config():
+    return {
+        "dataset": ThatiARDataset,
+        "task": SubjectivityTask,
+        "model": OpenAIModel,
+        "general_args": {"test_split": "ar/test"},
```

Comment:
In ThatiARDataset, we don't have "ar" split so I think this might fail, and you don't need to specify the test/train splits as the fraemwork will automatically find them.